applications/mne_analyze/extensions/music/music.h

```diff
@@ -0,0 +1,134 @@
+* PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+* INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+* PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+* HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+* NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+* POSSIBILITY OF SUCH DAMAGE.
+*
+*
+* @brief    Contains the declaration of the Music class.
+*
+*/
+
+#ifndef MUSIC_H
+#define MUSIC_H
+
+//*************************************************************************************************************
+//=============================================================================================================
+// INCLUDES
+//=============================================================================================================
+
+#include "music_global.h"
+
+#include <anShared/Interfaces/IExtension.h>
+
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// QT INCLUDES
+//=============================================================================================================
+
+#include <QtWidgets>
+#include <QtCore/QtPlugin>
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// FORWARD DECLARATIONS
+//=============================================================================================================
+
+class MusicControl;
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// DEFINE NAMESPACE DIPOLEFITEXTENSION
+//=============================================================================================================
+
+namespace MUSICEXTENSION
+{
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// FORWARD DECLARATIONS
+//=============================================================================================================
+
+
+//=============================================================================================================
+/**
+* Music Extension
+*
+* @brief The Music class provides input and output capabilities for the fiff file format.
```

Comment:
Iread this docu comment quite often now. Is this really io capability for the fiff file format? MUSIC != fiff